### DISCUSSION TITLE
Gas cost markers in remix

### DISCUSSION THREAD

**Discussion by getzelus** [Cyfrin/foundry-full-course-cu#617] [Q&A] [answered]
Hello, in the tutorial video at the beginning, 
we see the gas cost for each solidity function in Remix IDE. 

https://i.imgur.com/zLko3vB.jpg

But i dont see that in my browser. How can we activate this option ?

**Comment by 0xusmanf** [comment, marked as answer]
Click on the settings icon in bottom left corner of remix and enable the `Display gas estimates in editor.` check.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/65fe2260-b799-48e9-a163-8e179864d8d8)

**Reply by getzelus** [reply]
Thats great, thanks !
